__tests__/test-utils/hijack-del.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const del = require('del');
+
+// returns the `del` module but injects the `force=true`
+// option to allow for overwriting outside the <rootDir>
+module.exports = function() {
```

Comment:
This seems like a lot of trouble to avoid passing the `force: true` option. Could we just pass that option?

Reply:
That's true, in the latest commit I have changed the `del` calls to be forced.